packages/mui-material/src/styles/createTypography.js

```diff
@@ -69,7 +69,7 @@ export default function createTypography(palette, typography) {
     subtitle2: buildVariant(fontWeightMedium, 14, 1.57, 0.1),
     body1: buildVariant(fontWeightRegular, 16, 1.5, 0.15),
     body2: buildVariant(fontWeightRegular, 14, 1.43, 0.15),
-    button: buildVariant(fontWeightMedium, 14, 1.75, 0.4, caseAllCaps),
+    button: buildVariant(fontWeightMedium, 14, 1.5, 0.4, caseAllCaps),
```

Comment:
For reference, in Material Design 3, we would use 20/14 => 1.43 (and a different theming structure). 

<img width="1016" alt="Screenshot 2021-12-17 at 11 28 14" src="https://user-images.githubusercontent.com/3165635/146530756-0083fb97-e09c-4426-b2b3-b4965bb792da.png">
 
https://m3.material.io/styles/typography/tokens 

But I agree with @michaldudak, it's not light change.